app/lib/modules/login/view/login_view.dart

```diff
@@ -24,6 +24,10 @@ class LoginView extends StatelessWidget {
           ),
         ),
       ),
+
+      /// This code assumes that the AppRouter class is defined
+      /// elsewhere and contains a static member named signIn
+      /// that represents the route to navigate to.git
```

Comment:
Tuura ems tushundurup alganym uchun ayabai kechirim suraym. :(
```suggestion
      // SignIn sayfasını görmek için bunu active edebilirsiniz (tabi ingilizce olmalı)
      // floatingActionButton: FloatingActionButton(
      //   onPressed: () => Navigator.pushNamed(context, AppRouter.signIn),
      // ),
```